app/qna/llm.py

```diff
@@ -11,19 +11,27 @@
 
 INDEX_NAME = "wiki"
 
+OPENAI_API_TYPE = os.getenv("OPENAI_API_TYPE", "")
+OPENAI_COMPLETIONS_ENGINE = os.getenv("OPENAI_COMPLETIONS_ENGINE", "text-davinci-003")
 
 def create_vectorstore() -> Redis:
     """Create the Redis vectorstore."""
     import pandas as pd
 
-    from langchain.embeddings import OpenAIEmbeddings
 
-     # Init OpenAI Embeddings
-    oai_embeddings = OpenAIEmbeddings()
+    if OPENAI_API_TYPE=="azure":
```

Comment:
Can we pull this into a helper function like this?

`get_embeddings() -> Embddings`

I will need to reuse this for the semantic cache implementation coming soon on another branch I have

Reply:
done.